website/package.json

```diff
@@ -44,6 +44,7 @@
     "eslint-plugin-import": "2.14.0",
     "eslint-plugin-jsx-a11y": "6.1.2",
     "eslint-plugin-react": "7.11.1",
+    "firebase-mock": "2.2.10",
```

Comment:
Doesnt look like you are using this?

Reply:
I will keep them since we may use firebase-mock